editor/src/clj/internal/graph.clj

```diff
@@ -563,10 +563,15 @@
   (into '() (take-while some? (iterate (partial override-original basis) node-id))))
 
 (defn override-of [graph node-id override-id]
-  (some (fn [override-node-id]
-          (when (= override-id (gt/override-id (node-id->node graph override-node-id)))
-            override-node-id))
-        (overrides graph node-id)))
+  (let [^Indexed os (overrides graph node-id)
+        n (count os)]
+    (loop [i 0]
+      (if (= i n)
+        nil
+        (let [override-node-id (.nth os i)]
+          (if (= override-id (gt/override-id (node-id->node graph override-node-id)))
+            override-node-id
+            (recur (inc i))))))))
```

Comment:
This was an allocation hotspot because `some` created lazy sequences. I refactored it as a `loop`.

Reply:
Maybe we should have our own`coll/some`?

Is using a loop more efficient than `reduce`? 🤔

Reply:
Ah, fair enough!